### DISCUSSION TITLE
Work on docs coming along

### DISCUSSION THREAD

**Discussion by ParaplegicRacehorse** [LostRuins/koboldcpp#1175] [Show and tell]
It needs a whole lot, still, but it's coming along!

![image](https://github.com/user-attachments/assets/fe49e571-9724-439c-8738-7f6b56df4c5f)

![image](https://github.com/user-attachments/assets/4488194c-be5f-41bf-8220-6c18b31d07a6)

Much of this is gratuitously stolen from Concedo, reformatted, edited, redacted, misfiled, mailed to the Mars Rover, returned-to-sender, lost in a stack of USCG paperwork, buried in a pile of tabletop RPGs, forgotten for half-a-year, found, used as under-mat for miniatures painting, recycled into gerbil fill, rediscovered on a back-up HDD, dusted off, reformatted, edited, and now I'm sharing two screenshots of where things are currently at.

TODO:
- [ ] Write the _entire_ Glossary.
    - plagiarize others, then edit so they can't be certain that's what's been done
    - take credit for writing all
- [ ] Copy & edit the compile-from-source instructions
- [ ] Copy, rewrite, edit the instructions for acquiring model weights
- [ ] Write all the text for the GUI setup
- [ ] Better, longer, richer, more ~~foul~~ ... ~~colorful~~ ... _useful_ descriptions of the CLI options
- [ ] plagiarize, redact, edit how to connect clients to the APIs
- [ ] struggle with how to document koboldai-lite
    - [ ] write a terrible overview of all the settings pages
    - [ ] pretend that I wrote documentation and be done with it
- [ ] struggle with how to document sdwebui
    - [ ] write a terrible overview of the sliders and widgets
    - [ ] pretend this is proper documentation and be done with it
- [ ] write a terribly, wildly inaccurate but still somehow functional description of how to use kobold-cpp for training LLM finetunes 
- [ ] find a ~~schmuck~~ ... volunteer to create `favicon.png` and `favicon.svg`
- [ ] find a ~~loser~~ ... volunteer to create a better theme (, or several?)
- [ ] publish!
- [] claim credit for the whole of the work (after noting names in the contributors page
- update changelog at each stage

Note the un-ordered list; indicating that none of this will be done in the order presented.

**Comment by ParaplegicRacehorse** [comment]
No. This is not yet available anywhere public; not even for people who might have some masochistic desire to contribute. 
I'm sorry. 
You'll just have to let the boils on your butt fester for a while longer.

**Comment by ParaplegicRacehorse** [comment]
After some discussion with @henk717, this is being ported to his/her/their [koboldai.com](https://github.com/henk717/koboldai.com) repository. No commits from me, yet, but expect something soon-ish.

My updates will first land in my own repo [here](https://github.com/ParaplegicRacehorse/koboldai-docs)

**Comment by LostRuins** [comment]
btw for documentation, I maintain a project wiki which is the main reference which I will update, whenever anything is added or changed, found at https://github.com/LostRuins/koboldcpp/wiki , you can use that as a reference, if you want as everything related to koboldcpp is likely derived from this source. 

also, you might want to check the license from your screenshot - CC-BY-NA is not a valid thing. Did you mean CC-BY-NC? But that's also not the license of kobold, as that is under AGPLv3
